lib/applications/schema.ts

```diff
@@ -111,6 +112,7 @@ export default `
     rcdUserId: Int
     isRenewal: Boolean!
     permitType: PermitType!
+    receiveEmailUpdates: Boolean
```

Comment:
```suggestion
    receiveEmailUpdates: Boolean!
```
For new applications, this boolean should be non-nullable